utils/number.ts

```diff
@@ -0,0 +1 @@
+export const generateNegativeRandomNumber = () => Math.round(Math.random() * -1000000)
```

Comment:
```suggestion
export const generateNegativeRandomNumber = () => Math.round((Math.random() * -1000000) - 1)
```

Just me being pedantic really :p

Reply:
Great one! Since `Math.random` can generate a number
> in the range 0 to less than 1 (inclusive of 0, but not 1)
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random